libs/fills/src/lib/fills-table.tsx

```diff
@@ -479,20 +492,35 @@ export const getTotalFeesDiscounts = (fees: TradeFeeFieldsFragment) => {
 
 export const getFeesBreakdown = (
   role: Role,
-  feesObj: TradeFeeFieldsFragment
+  feesObj: TradeFeeFieldsFragment,
```

Comment:
It would be sensible to update the getFeesBreakdown tests to include a test checking for market state

Reply:
Added for each case!